py/torch_tensorrt/dynamo/_exporter.py

```diff
@@ -55,24 +48,27 @@ def transform(
 
     Returns an inlined torch.fx.GraphModule
     """
+    gm_export = copy.deepcopy(gm)
```

Comment:
Minor question - what is the motivation for not performing this function in-place?

Reply:
We deepcopy the graph here because the transformations in this function  as it lifts the graph's parameters/buffers/constants as inputs and the input gm becomes unusable if we perform in-place operation.